### PR TITLE
Migration from SVN to Git

### PR DESCRIPTION
**Pull request by MariaShaposhnikova**

### CODE REVIEW

**Comment by MariaShaposhnikova** [comment]
@tknippenberg , please review the PR.

**Review by tknippenberg — COMMENTED** [review]
> Creates a *local* commit: a set of changes with a message that you can jump back to. Changes are not submitted to the server, unless you check the **Push** checkbox.

consider changing to 

> Creates a *local* commit: a set of changes with a message that you can jump back to. Changes are not submitted to the server, unless the **Push** checkbox is checked.

**Review by tknippenberg — COMMENTED** [review]
These sentence feels a bit weird to me.
> Current version-controller is in Team Server SVN

Last part of the sentence feels weird.
> Inform your team members that they should commit their changes before migration has started not to lose their uncommitted changes.

**Review by tknippenberg — APPROVED** [review]
Overall looks good to me!

**Comment by tknippenberg** [comment]
@grootjans , if you like you can take another look. I don't think it's necessary per se.

**Comment by MariaShaposhnikova** [comment]
@tknippenberg , thank you for the review! I have rewritten sentences that you had comments on.

**Comment by MariaShaposhnikova** [comment]
@Luyao-Zhang-1 , could you please take a look if the Developer Portal part looks good to you?

File: content/en/docs/developerportal/collaborate/team-server/migrate-to-git.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+Git is a more modern version control system that has several advantages over SVN. For more information on advantages of Git, see the [What Are the Advantages of Team Server Git over Team Server SVN?](/refguide/version-control-faq/#git-advantages) section in *Version Control FAQ*.
+
+## 2 Apps Eligible for Migration
+
+To be able to migrate your app to Git, your app needs to meet the following criteria:
+
+* Mendix version of the app is 9.12 or above
+* Current version control is in Team Server SVN
+* App has maximum 100 revisions across all branches (we plan to increase this maximum the coming months)
```
Comment (Luyao-Zhang-1) [inline content/en/docs/developerportal/collaborate/team-server/migrate-to-git.md]:
```suggestion
* Your app has maximum 100 revisions across all branches (we plan to increase this maximum in the coming months)
```

**Review by Luyao-Zhang-1 — APPROVED** [review]

**Review by tknippenberg — CHANGES_REQUESTED** [review]
The "(Optional) Migration scheduled" email won't be sent yet as we don't have scheduling options yet. Can this still be removed?

**Comment by MariaShaposhnikova** [comment]
@tknippenberg , sure, I've delete this information.

**Pull request merged**